lighthouse/logos/README.md

```diff
@@ -0,0 +1,19 @@
+# Logos
+
+The logos folder should contain three images with filenames and sizes that exactly match the following specifications. Underneath each specification is a list of places where the images may appear in the web app.
+
+#### `saas_logos-bot.png` (200 × 128)
+
+- Integration tile images at `/account/settings`
+- Description heading at `/account/settings#integrations/lighthouse`
+- Integration monitor tiles and search bar results images at `/monitors#create/integration`
+
+#### `saas_logos-small.png` (120 × 60)
+
+- Integration dashboards list images at `/dash/list`
+- Some integration dashboards/screenboards at `/dash/integration/lighthouse`
+
+#### `avatars-bot.png` (128 × 128)
+
+- Event stream at `/event/stream`
+- Notification icons at `/report/monitor`
```

Comment:
Seems the issue is still there :) Here is the diff of the related Github commit.

![image](https://user-images.githubusercontent.com/49917914/59593005-6b1b9a80-90f1-11e9-9ff2-db742602454a.png)

Reply:
oy, ok I guess I am a bit confused? according to the readme doc and spot checking vs other integrations, these pixel width/height seem correct to me? ie
`saas_logos-small.png`: 120px Width / 60px Height
`saas_logos-bot.png`: 200px Width / 128px Height
`avatars-bot.png`: 128px Width / 128px Height

Reply:
Sorry, I wasn't clear on my previous comment. The size fine, but the images seems to be squeezed horizontally or vertically somehow, not 1:1 scaling.

Reply:
ahh, i see, apologies on that my eye for visual is poor. I found some assets that scaled a little more appropriately, pushed them up in last commit, the squeezed effect should be gone if you want to take a look. Thanks again appreciate the patience here.

Reply:
`lighthouse/logos/saas_logos-small.png` seems still a bit squeezed :)

![image](https://user-images.githubusercontent.com/49917914/59682060-b4ddb100-91d5-11e9-9f6e-49b633d112a4.png)

Reply:
oy, alrighty i updated again, i am almost positive this one is pixel perfect